lib/src/git.rs

```diff
@@ -1909,6 +1909,7 @@ pub fn add_remote(
     remote_name: &RemoteName,
     url: &str,
     fetch_tags: gix::remote::fetch::Tags,
+    refspec_patterns: Option<&[StringPattern]>,
```

Comment:
nit: same here. rename to `branch_names` or `branch_patterns`?

Reply:
Good point